### PR TITLE
fix: Remove double tooltip from metric label

### LINKED ISSUE #12671: [explore] Tooltip displays twice for Metric
## Screenshot
Issue 1: metric definision (logic) displays twice
Screen Shot 2021-01-21 at 5 06 30 PM

Issue 2:metric description (text) displays over the definition (logic) - probably the logic should not appear in this case
Screen Shot 2021-01-21 at 5 08 12 PM

## Description

On chart Explore, sometimes the tooltip - metric definition (logic) - displays twice - I noticed it when user moves mouse from left to right. Another issue, if the metric definition (logic) displays and then user hovers over the (i) to see description, the description displays over the metric definition. Shouldn't the definition in this case disappear?

## Design input
Straightforward fix I believe as described in description

### PR DESCRIPTION
**Pull request by kgabryje**
### SUMMARY
TODO: After https://github.com/apache-superset/superset-ui/pull/914 is merged and released, bump superset-ui packages in this PR.
After packages are bumped, this PR should be tested thoroughly as the change in `superset-ui` affects multiple components that used `InfoTooltipWithTrigger` component.
Fixes https://github.com/apache/superset/issues/12671

### ADDITIONAL INFORMATION

- [x] Has associated issue: https://github.com/apache/superset/issues/12671
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

CC: @villebro @junlincc

### CODE REVIEW

**Comment by junlincc** [comment]
thanks for the PR! @kgabryje what about showing truncated label?

**Comment by villebro** [comment]
This looks great! A few observations:
- When using Tab to commit a value to the IN filter, the focus shifts to the Save button: 
sf

- For saved metrics this seems to work as expected, but for simple and custom metrics the tooltip doesn't appear: 
simple

- Now that we only display a single tooltip, I wonder if it would be easier on the eyes to have both tooltips on the top?
tooltip

**Comment by kgabryje** [comment]
@villebro 1. In progress, 2. Addressed in this PR, 3. Addressed in https://github.com/apache-superset/superset-ui/pull/914

**Comment by kgabryje** [comment]
1. I'm not sure we can do anything about it - `antd Select` doesn't expose any API for handling such case and `event.stopPropagation()`on `onKeyPress/Up/Down` events doesn't fix the issue.

**Review by villebro — APPROVED** [review]
Thanks @kgabryje - Given that having tab change focus is not a regression (AFAIK), but rather an improvement on the previous functionality, I think this is fine as-is. However, since `superset-ui` is now on `v0.17.0` and we're still waiting for #10270 to be merged to bump to that version, we need to hold off on merging this PR until that one gets merged. So once that's merged, let's merge https://github.com/apache-superset/superset-ui/pull/914, deploy that and bump `superset-ui/chart-controls` to the new version in this PR.

**Pull request merged**
